inscriptions/ISic000683.xml

```diff
@@ -191,11 +196,17 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="0"/><gap reason="lost" quantity="1" unit="line"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">optim</supplied>o <g ref="#interpunct">·</g> <name><expan><abbr>Sex</abbr><ex>ti</ex></expan></name> <name><seg part="I">Se<unclear>l</unclear></seg></name> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">pe</supplied>cunia <g ref="#interpunct">·</g> eius <g ref="#interpunct">·</g> <unclear>c</unclear><supplied reason="lost">olonis</supplied> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="3"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">epulum</supplied> <expan><abbr><supplied reason="lost">si</supplied>ng</abbr><ex>ulis</ex></expan> <gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/><gap reason="illegible" quantity="2" unit="character"/><gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
? there's no fourth line - you seem to have confused this with line 3, (which only has traces of NG on it)